zppy/logger.py

```diff
@@ -0,0 +1,67 @@
+"""Logger module for setting up a logger. Based on xcdat/xcdat/_logger.py"""
```

Comment:
This is redundant with https://github.com/E3SM-Project/zppy-interfaces/blob/initial-implementation/zppy_interfaces/multi_utils/logger.py but I think that's pretty hard to avoid.

Still, worth keeping an eye on this kind of redundancy and having a backup plan if it gets bad.